src/app/application.h

```diff
@@ -103,6 +106,10 @@ private slots:
     QPointer<WebUI> m_webui;
 #endif
 
+    // FileLog
+    bool m_fileLogEnabled;
+    QPointer<FileLogger> m_fileLogger;
```

Comment:
We do cleanup in Application::cleanup() instead of destructor. The reason is we need to control the shutdown process on Windows otherwise windows kills us aka unclean exit.